target/linux/ath79/image/mikrotik.mk

```diff
@@ -35,3 +35,19 @@ define Device/mikrotik_routerboard-wap-g-5hact2hnd
   SUPPORTED_DEVICES += rb-wapg-5hact2hnd
 endef
 TARGET_DEVICES += mikrotik_routerboard-wap-g-5hact2hnd
+
+define Device/mikrotik_sxt5n
+  $(Device/mikrotik)
+  SOC := ar9344
+	DEVICE_MODEL := SXT5N platform
+  IMAGE/sysupgrade.bin = append-kernel | kernel2minor -s 2048 -e -c | \
+	sysupgrade-tar kernel=$$$$@ | append-metadata
+  DEVICE_PACKAGES += nand-utils rssileds kmod-gpio-beeper
+  SUPPORTED_DEVICES += rb-sxt5n
+endef
+
+define Device/mikrotik_rbsxt5ndr2
```

Comment:
We don't have that abbreviated product code elsewhere. So either, we could use the longer version
mikrotik_routerboard-sxt-5ndr2
or just take the
mikrotik_sxt-lite5
if that's sufficiently unique.

Reply:
You're right.

I like the "mikrotik_routerboard-sxt-5ndr2", which better adheres to MikroTik's device identification scheme. Actually, since both the sticker on the device and `hard_config` report "SXT 5nD r2", I'll use "mikrotik_routerboard-sxt-5ndr2".

By the way, I did a quick search and it doesn't look like there was an "r1" version, so the "r2" would not indicate "version 2" of sthg.

I'm renaming the device model to `DEVICE_MODEL := SXT Lite5 (RouterBOARD SXT 5nD r2)`, which I believe it's more readable. I also think the "wAP AC" model should be written the other way round (`DEVICE_MODEL := wAP AC (RouterBOARD wAP G-5HacT2HnD`). The 922UAGS-5HPacD is OK as it is (it doesn't have a commercial name different to the product code).

Reply:
Changed (mikrotik,rbsxt5ndr2 => mikrotik,routerboard-sxt-5nd-r2).